dev/io.openliberty.faces.fat.selenium.util.internal/.settings/org.eclipse.jdt.ui.prefs

```diff
@@ -0,0 +1,116 @@
+#Ant properties
```

Comment:
Can you compare this to the EE11 projects for WebTier the copyright etc looks incorrect in this file. Let's make sure the .settings are consistent.